### PR TITLE
chore: 🍰 Replace node-sass with sass

### LINKED ISSUE #4068: 💥 [DevOps] node-sass is deprecated
## :fire: DevOps ticket
We should use dart-sass

### Motive
A lot of bumps fail because of errors caused by node-sass. Checking the page of the package, says it is deprecated:
https://sass-lang.com/blog/libsass-is-deprecated

### Related issues
Tons of bumps

### Implementation
If you’re a user of Node Sass, migrating to Dart Sass is straightforward: just replace node-sass in your package.json file with sass. Both packages expose the same JavaScript API.

This is what the page says.

### Validation
Be careful

### PR DESCRIPTION
**Pull request by Mogge**
## 🍰 Pullrequest
As node-sass is deprecated it is recommended to use dart-sass instead. We had a lot of build problems with node-sass. Hopefully these are gone now.

### Issues
- fixes #4068

### CODE REVIEW

**Review by Tirokk — COMMENTED** [review]
Hey, @mogge In general everything looks good so far. 💫🤗

But `node-sass` seems still included in the web apps `package.json` and seems as well to be used in the styleguide.

What can we do about the styleguide?

File: webapp/yarn.lock [hunk 1]
```diff
@@ -12523,7 +12538,7 @@ node-res@^5.0.1:
     on-finished "^2.3.0"
     vary "^1.1.2"
 
-node-sass@^4.12.0, node-sass@~4.13.1:
+node-sass@^4.12.0:
```
[the remark this answers is not in the document]
Comment (Mogge) [reply webapp/yarn.lock:12541]: I do not know. What is the purpose of the yarn.lock file?

[the remark this answers is not in the document]
Comment (Mogge) [reply webapp/yarn.lock:12541]: This is the yarn.lock file.

[the remark this answers is not in the document]
Comment (Mogge) [reply webapp/yarn.lock:12541]: The styleguide is imported as an npm package. So I see no problem here

**Review by ulfgebhardt — APPROVED** [review]
lets get this merged

**Pull request merged**